fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -771,6 +771,18 @@ Function getDecimalV3Function(FunctionCallExpr node, Type[] argumentTypes) {
                         node.getParams().isStar() ? "*" : Joiner.on(", ")
                                 .join(Arrays.stream(argumentTypes).map(Type::toSql).collect(Collectors.toList())));
             }
+            
+            
+            if (FunctionSet.MAX_BY.equals(fnName)) {
+                if (argumentTypes[0].isDecimalV3()) {
```

Comment:
should handle in DecimalV3FunctionAnalyzer

Reply:
fixed